gravityflow/gravityflow
# Add setting to enable the Workflow Inbox Count

## Description
This PR adds  a new plugin setting to control the display of the Workflow Inbox Count and removes the `gravityflow_inbox_count_display` filter added in #354. This setting appears in. a new "Advanced" section and the security settings have been moved into this new section.


## Testing instructions
1. Update to this branch
2. Check the Workflow Settings, has the option 'Workflow Inbox Count' added.
3. Update to On/Off to check if the settings work as desired.
4. Test on both Gravity Forms 2.4 and 2.5.


## Documentation Changes?
This feature will need documenting.


## Checklist:
